README.md

```diff
@@ -6,7 +6,7 @@ This plugin allows you to censor profanity on your Mattermost server.
 
 ## Installation
 
-1. Go to the [releases page of this Github repository](https://github.com/mattermost/mattermost-plugin-antivirus/releases) and download the latest release for your Mattermost server.
+1. Go to the [releases page of this Github repository](https://github.com/mattermost/mattermost-plugin-profanity-filter) and download the latest release for your Mattermost server.
```

Comment:
Would you please keep the link to the releases page?

Reply:
Done :)